Resources/Prototypes/Floof/Entities/Mobs/NPCs/Xeno.yml

```diff
@@ -0,0 +1,13 @@
+- type: entity
+  name: Friendshaped
+  parent: MobXenoRunnerNPC
+  id: MobFriendshapedNPC
+  components:
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+    - id: FoodMeatRouny
+      amount: 3
```

Comment:
This change should be given to MobXenoRunnerNPC instead.

Reply:
it's set the way it is because that's how it's set up on the regular Rouny npc. 

I suppose just parenting him to the Rouny NPC would work better than copying the parent of the Rouny NPC

Reply:
I can go ahead and do that now

Reply:
done.